beets/ui/__init__.py

```diff
@@ -1261,6 +1266,8 @@ def _raw_main(args, lib=None):
                       help='path to configuration file')
     parser.add_option('-p', '--plugins', dest='plugins',
                       help='a comma-separated list of plugins to load')
+    parser.add_option('-P', '--disable-plugin', dest='exclude',
```

Comment:
Argh, sorry for the bad advice!! It should be `--disable-plugins`, i.e., symmetric with the "positive" flag name.